ui/src/submissions/authors/containers/AuthorSubmissionPageContainer.jsx

```diff
@@ -26,18 +26,29 @@ class AuthorSubmissionPage extends Component {
   }
 
   render() {
-    const { error, query } = this.props;
+    const { error, query, profileControlNumber } = this.props;
     const { bai } = query;
     const initialFormData = { bai };
+    const authorUpdateUrl = `${SUBMISSIONS_AUTHOR}/${profileControlNumber}`;
     return (
       <SubmissionPage
         title="Suggest author"
         description={
-          <span>
-            This form allows you to create the profile of a new author. It will
-            be added to the <Link to={AUTHORS}>authors collection</Link> upon
-            approval.
-          </span>
+          <>
+            <span>
+              This form allows you to create the profile of a new author. It
+              will be added to the <Link to={AUTHORS}>authors collection</Link>{' '}
+              upon approval.
+            </span>
+            {profileControlNumber && (
+              <p className="mt-4">
+                <strong>
+                  Do you want to update your profile? Go to the{' '}
+                  <Link to={authorUpdateUrl}>author update form.</Link>
```

Comment:
nit: very small improvement, lets put the . after the link. So the link is only on author update form